client/src/preview/util/fileHandler.ts

```diff
@@ -50,7 +50,7 @@ class FileHandler implements FileHandlerInterface {
     await this.backend.api.createRepositoryFile(
       projectToUse,
       `${filePath}/${file.name}`,
-      'main',
+      'master',
```

Comment:
This constant literal is used in many places. It might be best to just make it a setting as well.